Mathlib/Analysis/SpecialFunctions/Gamma/Beta.lean

```diff
@@ -334,7 +334,8 @@ theorem approx_Gamma_integral_tendsto_Gamma_integral {s : ℂ} (hs : 0 < re s) :
       Measure.restrict_restrict_of_subset Ioc_subset_Ioi_self]
   -- f is uniformly bounded by the Gamma integrand
   · intro n
-    refine' (ae_restrict_iff' measurableSet_Ioi).mpr (ae_of_all _ fun x hx => _)
+    refine' (ae_restrict_iff' measurableSet_Ioi).mpr ?_
```

Comment:
`rw [ae_restrict_iff' measurableSet_Ioi]`

Reply:
This worked and is even nicer.